go-vulncheck/action.yml

```diff
@@ -24,6 +24,7 @@ runs:
       with:
         go-version-file: 'go.mod'
         go-version: '1.21.3'
```

Comment:
i think you can remove go-version with this change @pkramaric

Reply:
i went with this https://github.com/actions/setup-go#check-latest-version. But I am not sure if removing will do the latest go version i am okay with that, but according to doc in the link it says the default is false for check new version.